src/Microsoft.DotNet.Darc/src/DarcLib/Commit.cs

```diff
@@ -6,13 +6,16 @@ namespace Microsoft.DotNet.DarcLib
 {
     public class Commit
     {
-        public Commit(string author, string sha)
+        public Commit(string author, string sha, string message)
         {
             Author = author;
             Sha = sha;
+            Message = message;
+
         }
 
-        public string Author { get; }
-        public string Sha { get; }
+        public string Author { get; set; }
```

Comment:
Is there a strong reason to make this class mutable? Immutable classes are a lot easier to not introduce bugs in.

Reply:
Yes, I need to store the commit information and read it later on during the time of merge so I have to make it mutable.

Reply:
Why not just make a new copy of the object with the right value?  I don't recall seeing any setters of this stuff, but I might have missed it.

Reply:
I synced this branch and removed the setters, and it compiles without issue.  Why add setters and make this mutable when the code doesn't require that?